src/containers/PageBuilder/Primitives/SearchCTA/FilterDateRange/FilterDateRange.js

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+
+import { useIntl } from '../../../../../util/reactIntl';
+
+import { OutsideClickHandler, IconDate, FieldDateRangeController } from '../../../../../components';
+
+import css from './FilterDateRange.module.css';
+
+/**
+ * FilterDateRange displays a toggleable date range picker.
+ *
+ * @component
+ * @param {Object} props Component properties.
+ * @param {Object} props.config Marketplace configuration object
+ * @param {string?} props.className e add more style rules in addition to components own css.root
+ * @param {string?} props.rootClassName overwrite components own css.root
+ * @returns {JSX.Element} The FilterDateRange component.
+ */
+const FilterDateRange = props => {
+  const [isOpen, setIsOpen] = useState(false);
+  const [selectedDates, setSelectedDates] = useState(null);
+  const { className, rootClassName, config } = props;
+  const intl = useIntl();
+
+  const classes = classNames(rootClassName || css.root, className);
+
+  const formatDateRange = (start, end) => {
+    const formattedStart = intl.formatDate(start, { day: 'numeric', month: 'short' });
+    const formattedEnd = intl.formatDate(end, { day: 'numeric', month: 'short' });
+    return `${formattedStart} - ${formattedEnd}`;
```

Comment:
I don't think we can use this format
![Screenshot 2025-04-23 at 22 27 45](https://github.com/user-attachments/assets/f6cdfc78-07f8-4f73-8cb8-83d2cb94eb54)

I guess the options are 
- set both day & month to numeric, but format them separately
  - fi: **10.1. - 12.1.**, en-US: **1/10 - 1/12**, en-GB: **10/01 - 12/01**
- Use [formatDateTimeRange](https://formatjs.github.io/docs/react-intl/api#formatdatetimerange)
  - This should look like this with month set to 'short': 
    fi: **10.–12. tammikuuta**, en-US: **Jan 10 – 12**, en-GB: **10–12 Jan**
  - or as both numeric: 
    fi: **10.–12.1.**, en-US: **1/10 – 1/12**, en-GB: **10/01 – 12/01**

I think all of these should fit to the space we have available.

Reply:
@shareoc perhaps you could escalate this to product channel and ping e.g. juho there?
I'm leaning towards second option (fi: 10.–12. tammikuuta, en-US: Jan 10 – 12, en-GB: 10–12 Jan)

Reply:
@Kusti what do you think?

Reply:
@shareoc @Gnito the second option looks like an obvious choice to me too, let's go with that!